### ISSUE TITLE
Train-accuracy=0.000000

### ISSUE THREAD

**Issue by BrettLL** [apache/incubator-mxnet#5326]
When I finetune the resnet50 with a small dataset(several thousand), the train accuracy can achieve above 90% fast, but when I use the same net and configuration to finetune resnet50 with a big dataset(millions), the train accurancy and the valid accurancy are always 0.00000.

The log is as follows:
 multi_factor_scheduler(begin_epoch, epoch_size, step=[40, 60], factor=0.1),
/home/hikvision/mxnet/python/mxnet/model.py:516: DeprecationWarning: [91mCalling initializer with init(str, NDArray) has been deprecated.please use init(mx.init.InitDesc(...), NDArray) instead.[0m
  self.initializer(k, v)
INFO:root:Start training with [gpu(0), gpu(1), gpu(2), gpu(3), gpu(4), gpu(5), gpu(6), gpu(7)]
[19:11:43] src/operator/./cudnn_convolution-inl.h:55: Running performance tests to find the best convolution algorithm, this can take a while... (setting env variable MXNET_CUDNN_AUTOTUNE_DEFAULT to 0 to disable)
INFO:root:Epoch[0] Batch [50]	Speed: 129.39 samples/sec	Train-accuracy=0.002266
INFO:root:Epoch[0] Batch [50]	Speed: 129.39 samples/sec	Train-top_k_accuracy_5=0.003281
INFO:root:Epoch[0] Batch [100]	Speed: 127.94 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [100]	Speed: 127.94 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [150]	Speed: 128.39 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [150]	Speed: 128.39 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [200]	Speed: 128.75 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [200]	Speed: 128.75 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [250]	Speed: 128.73 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [250]	Speed: 128.73 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [300]	Speed: 128.83 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [300]	Speed: 128.83 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [350]	Speed: 129.20 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [350]	Speed: 129.20 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [400]	Speed: 128.69 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [400]	Speed: 128.69 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [450]	Speed: 126.95 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [450]	Speed: 126.95 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [500]	Speed: 126.86 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [500]	Speed: 126.86 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [550]	Speed: 128.59 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [550]	Speed: 128.59 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [600]	Speed: 129.08 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [600]	Speed: 129.08 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [650]	Speed: 129.52 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [650]	Speed: 129.52 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [700]	Speed: 129.66 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [700]	Speed: 129.66 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [750]	Speed: 129.70 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [750]	Speed: 129.70 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [800]	Speed: 128.52 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [800]	Speed: 128.52 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [850]	Speed: 129.81 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [850]	Speed: 129.81 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [900]	Speed: 129.08 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [900]	Speed: 129.08 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [950]	Speed: 129.71 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [950]	Speed: 129.71 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1000]	Speed: 129.39 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1000]	Speed: 129.39 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1050]	Speed: 129.74 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1050]	Speed: 129.74 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1100]	Speed: 130.68 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1100]	Speed: 130.68 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1150]	Speed: 131.13 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1150]	Speed: 131.13 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1200]	Speed: 131.57 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1200]	Speed: 131.57 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1250]	Speed: 131.76 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1250]	Speed: 131.76 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1300]	Speed: 130.42 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1300]	Speed: 130.42 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1350]	Speed: 130.49 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1350]	Speed: 130.49 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1400]	Speed: 129.35 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1400]	Speed: 129.35 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1450]	Speed: 132.04 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1450]	Speed: 132.04 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1500]	Speed: 130.21 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1500]	Speed: 130.21 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1550]	Speed: 131.17 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1550]	Speed: 131.17 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1600]	Speed: 130.96 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1600]	Speed: 130.96 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1650]	Speed: 133.14 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1650]	Speed: 133.14 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1700]	Speed: 134.15 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1700]	Speed: 134.15 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1750]	Speed: 131.56 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1750]	Speed: 131.56 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1800]	Speed: 130.91 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1800]	Speed: 130.91 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1850]	Speed: 131.82 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1850]	Speed: 131.82 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1900]	Speed: 131.28 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1900]	Speed: 131.28 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [1950]	Speed: 132.99 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [1950]	Speed: 132.99 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2000]	Speed: 133.99 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2000]	Speed: 133.99 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2050]	Speed: 133.46 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2050]	Speed: 133.46 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2100]	Speed: 134.93 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2100]	Speed: 134.93 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2150]	Speed: 134.19 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2150]	Speed: 134.19 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2200]	Speed: 134.15 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2200]	Speed: 134.15 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2250]	Speed: 132.60 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2250]	Speed: 132.60 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2300]	Speed: 132.70 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2300]	Speed: 132.70 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2350]	Speed: 132.91 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2350]	Speed: 132.91 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2400]	Speed: 134.56 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2400]	Speed: 134.56 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2450]	Speed: 132.99 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2450]	Speed: 132.99 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2500]	Speed: 134.53 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2500]	Speed: 134.53 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2550]	Speed: 133.69 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2550]	Speed: 133.69 samples/sec	Train-top_k_accuracy_5=0.000000
INFO:root:Epoch[0] Batch [2600]	Speed: 132.43 samples/sec	Train-accuracy=0.000000
INFO:root:Epoch[0] Batch [2600]	Speed: 132.43 samples/sec	Train-top_k_accuracy_5=0.000000
...
By the way, i use the the work from https://github.com/tornadomeet/ResNet.

**Comment by BigFishMaster** [comment]
Have you solved this problem? I have the same problem...

**Comment by edmBernard** [comment]
welcome in deeplearning ^^
You have to adjust learning rate and/or dataset size. 
Do you start from a pretrained network or from random initialised weight ?

**Comment by yimyom** [comment]
any hope to have the solution to this problem ?
even by using other datasets/learning rate/etc... I keep having exactly a train-accuracy = 0

**Comment by edmBernard** [comment]
@yimyom what kind of network did you use ? dataset ? did you use pretrained model ?
from my personnal experience to finetune model try a small learning rate for 2/3 iterations and after use a more regular learning rate. If you have enough processing power you can try hyperparameter space exploration to find the better parameters.

**Comment by yimyom** [comment]
I'm building an autoencoder. I found my bug in fact. I was using an "accuracy" loss function instead of a "mse" (or "mae" or "rmse"). For those making the same mistake as me, "accuracy" is used to test for classification and you want the output of the network to be exactly the label of your classe. Obviously in a regression problem, the result will never be exactly the same as the observed value. So you want to use mse (or rmse or mae) instead because they compute a squared-error (root squared-error, absolute error).
@edmBernard  : thanks for the advice. I will indeed add a phase of layer-wise pretraining. It seems it will improve the model by a lot.

**Comment by eric-haibin-lin** [comment]
FYI there's an example for layer-wise pretrain for auto-encoder 
https://github.com/apache/incubator-mxnet/blob/master/example/autoencoder/autoencoder.py

**Comment by szha** [comment]
@apache/mxnet-committers: This issue has been inactive for the past 90 days. It has no label and needs triage.

For general "how-to" questions, our [user forum](https://discuss.mxnet.io/) (and [Chinese version](https://discuss.gluon.ai/)) is a good place to get help.

**Comment by vdantu** [comment]
@BrettLL : Is this issue solved ?

**Comment by nswamy** [comment]
Please post questions on our user-forum. Closing this now.

**Comment by cjolivier01** [comment]
Forum: https://discuss.mxnet.io/
